### PR TITLE
Clean up of text in Remix UI

### PR DESCRIPTION
**Pull request by greggdourgarian**

### CODE REVIEW

**Comment by yann300** [comment]
Thanks for this PR!

File: src/app/contract-tab.js [hunk 1]
```diff
@@ -87,21 +87,18 @@ function contractTab () {
...
       <div class="${css.crow}">
-        <div class="${css.col1_1}">Transaction gas limit</div>
+        <div class="${css.col1_1}">Gas limit</div>
         <input type="number" class="${css.col2}" id="gasLimit" value="3000000">
       </div>
       <div class="${css.crow} hide">
       <div class="${css.col1_1}">Gas Price</div>
         <input type="number" class="${css.col2}" id="gasPrice" value="0">
       </div>
-      <div class="${css.crow}">
```
Comment (yann300) [inline src/app/contract-tab.js]: why deleting this field?

File: src/app/staticanalysis/modules/txOrigin.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-var name = 'tx.origin: '
+var name = 'Origin: '
```
Comment (yann300) [inline src/app/staticanalysis/modules/txOrigin.js]: `transaction origin` ?

Comment (greggdourgarian) [inline src/app/staticanalysis/modules/txOrigin.js]: Hi Yann,  Yes, transaction origin, which is how it is described after the lead "Origin: ".   This change is cleaner and leaves no ambiguity about the field being tx.origin.

Comment (yann300) [inline src/app/staticanalysis/modules/txOrigin.js]: hi, I meant that I would just name it `Transaction origin` instead of `Origin`

**Comment by greggdourgarian** [comment]
Hi Yann

Yes, sure.    That works!

________________________________
________________________________

**Pull request closed without merging**